sensor/docs/tutorials/tag_based_pnp_calibrator.md

```diff
@@ -0,0 +1,153 @@
+# tag_based_pnp_calibrator
+
+In this tutorial, we present a hands-on tutorial of the `tag_based_pnp_calibrator`. Although we provide pre-recorded rosbag, the flow of the tutorial is meant to show the user the steps they must perform in their own use cases with live sensors.
```

Comment:
`a pre-recorded rosbag`
Only one right?

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/cc97c92e86a0232697252ed721052b50e73c88a1